filebeat/module/system/module.yml

```diff
@@ -0,0 +1,12 @@
+dashboards:
```

Comment:
@tsg @monicasarbu Should we call this file `dashboards.yml`? I was at first confused what this file is about.